package.json

```diff
@@ -83,8 +84,8 @@
 		"should-semantic-release": "^0.2.1",
 		"typescript": "^5.2.2",
 		"vitest": "^0.34.6",
-		"yaml-eslint-parser": "^1.2.2",
-		"zip-lib": "^0.7.3"
+		"web-ext": "^7.8.0",
```

Comment:
Maybe something to look into later on...

```plaintext
 WARN  3 deprecated subdependencies found: har-validator@5.1.5, request@2.88.2, uuid@3.4.0
Already up to date
Progress: resolved 989, reused 947, downloaded 0, added 0, done
 WARN  Issues with peer dependencies found
.
└─┬ web-ext 7.8.0
  └─┬ addons-linter 6.13.0
    └─┬ addons-scanner-utils 9.3.0
      └── ✕ unmet peer node-fetch@2.6.11: found 3.3.1 in web-ext
```

Reply:
For the deprecation warnings: https://github.com/mozilla/sign-addon/issues/932.

For `node-fetch`: https://github.com/mozilla/addons-scanner-utils/issues/271.

Reply:
Aha! Nice. https://github.com/mozilla/web-ext/issues/2737

This will all be fixed soon.